### DISCUSSION TITLE
How to change the default directory location of payloads

### DISCUSSION THREAD

**Discussion by TI4NLI4NG** [projectdiscovery/nuclei#4018] [Q&A]
When using template files that require loading payload files, the payload parameter will point to a file in a helper directory. The helpers directory defaults to the "/root/nuclei-templates/helpers/" directory. Is there a parameter that can specify the address of this directory

**Comment by ehsandeep** [comment]
@TI4NLI4NG you can use `-lfa` option to enable loading payload file from any location.

```console
   -lfa, -allow-local-file-access  allows file (payload) access anywhere on the system
```

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
